### PR TITLE
Transfer TLSConfig from roundtripper in websocket dialer

### PR DESCRIPTION
**Pull request by juliens**
related to containous/traefik#2017

### CODE REVIEW

**Review by ldez — APPROVED** [review]
LGTM

File: forward/fwd.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ func New(setters ...optSetter) (*Forwarder, error) {
 	if f.httpForwarder.roundTripper == nil {
 		f.httpForwarder.roundTripper = http.DefaultTransport
 	}
+	f.websocketForwarder.TLSClientConfig = f.httpForwarder.roundTripper.(*http.Transport).TLSClientConfig
```
Comment (emilevauge) [inline forward/fwd.go:131]: ~~:warning:  `f.httpForwarder.roundTripper` can be `nil`~~

EDIT: nope...

Comment (juliens) [reply forward/fwd.go:131]: `http.DefaultTransport` can be `nil` ?

Comment (emilevauge) [reply forward/fwd.go:131]: Sorry, read too quickly, I edited my comment just before you answered ^^

**Review by emilevauge — APPROVED** [review]

**Review by m3co-code — APPROVED** [review]
LGTM 👍

**Review by nmengin — APPROVED** [review]
LGTM

**Pull request merged**